utils/das_data_sorter.py

```diff
@@ -128,43 +129,97 @@ def gps_data(self, data:pd.Series) -> pd.Series:
         0 for when GPS was turned off, 1 for when turned on.
         '''
         new_gps_data = []
+
         for index_array in self.indexes:
             if 1 in data[index_array].values:
                 new_gps_data.append(1)
             else:
                 new_gps_data.append(0)
-        
+
         return new_gps_data
     
+    def smooth_caller(self, n:int, technique:str) -> None:
```

Comment:
Again, `_caller` is probably not necessary in this function name. I think you might have accidentally readded it when you combined the two functions into one haha

Reply:
Should this be changed to `smooth`?

Reply:
I would also remove the `_caller` bit to the method name. As you use more and more libraries in various languages, you will learn how to call your method names. As a developer, it would be a bit weird for me to add on the `_caller` bit because I don't care if it's a function that calls another function. I just want it to smooth my data! So yes, `smooth` would be a good method name for this!